policy/modules/system/systemd.te

```diff
@@ -1144,6 +1144,64 @@ optional_policy(`
 	dbus_watch_system_bus_runtime_named_sockets(systemd_resolved_t)
 ')
 
+#########################################
+#
+# Socket-proxyd local policy
+#
+## <desc>
+##  <p>
+##  Allow systemd-socket-proxyd to bind any port instead of one labelled
+## with systemd_socket_proxyd_port_t.
+##  </p>
+## </desc>
+gen_tunable(systemd_socket_proxyd_bind_any, false)
+
+## <desc>
+## <p>
+## Allow systemd-socket-proxyd to connect to any port instead of
+## labelled ones.
+## </p>
+## </desc>
+gen_tunable(systemd_socket_proxyd_connect_any, false)
+
+type systemd_socket_proxyd_t;
+type systemd_socket_proxyd_exec_t;
+init_daemon_domain(systemd_socket_proxyd_t, systemd_socket_proxyd_exec_t)
+
+kernel_read_system_state(systemd_socket_proxyd_t)
+
+auth_use_nsswitch(systemd_socket_proxyd_t)
+sysnet_dns_name_resolve(systemd_socket_proxyd_t)
+
+type systemd_socket_proxyd_unit_file_t;
+init_unit_file(systemd_socket_proxyd_unit_file_t)
+
+# resolver
+allow systemd_socket_proxyd_t self:unix_dgram_socket { create getopt setopt sendto read write };
+
+# listener
+type systemd_socket_proxyd_port_t;
+corenet_port(systemd_socket_proxyd_port_t)
```

Comment:
Is the expectation that the user always has to set a particular port? there's not even one standard port?

Reply:
Yes, this is the expectation.  Due to the nature of the daemon (it is a proxy) there is no default allocated standard port -- a user needs to determine what they want to proxy and where.  The reason why I introduced port labelling here is that some administrators may want to restrict which ports they would allow the daemon to operate on.

P.S. You did not comment to `type systemd_socket_proxyd_port_t` to be moved to the declaration section, but after looking at the current structure I found that it is likely what you may want, so I am moving that type definition to the top too.